version: '3.5'
services:
  chunk_manager:
    container_name: chunk_manager
    image: chunk_manager:latest
    entrypoint: python3 /main.py
    restart: on-failure
    environment:
      - PYTHONUNBUFFERED=1
      - QUEUE_MAP=map_city
      - QUEUE_COUNT=count_controller_worker
      - QUEUE_DATE=date_redirector_worker
      - EOF_MAP=eof_map
      - EOF_COUNT=eof_count
      - EOF_DATE=eof_date
      - TOPIC_PLACES=places
    volumes:
      - ./chunk_manager/data:/data
    networks:
      - test-network      

networks:
  test-network:
    external:
      name: ${SERVER_NAME}-network